tests/ui_tests/test_configs.py

```diff
@@ -62,8 +66,28 @@ def _generate_fields():
     return fields
 
 
+def cluster_with_service(sdk_client: ADCMClient, path) -> Cluster:
+    """
+    Prepared cluster for test: create cluster, couple services and couple components.
+    """
+    bundle = sdk_client.upload_from_fs(path)
+    cluster_name = "_".join(path.split("/")[-1:] + [random_string()])
```

Comment:
If there's a need for a complex name, maybe it should be in utils

Reply:
no need, just took it from the previous code block